packages/labs/react/src/create-component.ts

```diff
@@ -127,13 +135,13 @@ type EventProps<R extends Events> = {
  * messages. Default value is inferred from the name of custom element class
  * registered via `customElements.define`.
  */
-export const createComponent = <I extends HTMLElement, E extends Events>(
-  React: typeof ReactModule,
-  tagName: string,
-  elementClass: Constructor<I>,
-  events?: E,
-  displayName?: string
-) => {
+export const createComponent = <I extends HTMLElement, E extends Events>({
```

Comment:
Let's make this new signature an additive and non-breaking change by duck-typing. Keep the old signature as an override and add the new signature as an override. Make the base function signature take a `typeof ReactModule | CreateComponentParams<I, E>` as the first argument called `optionsOrReact`, and `tagName?`, etc as the remaining argument. If `tagName` is a string, assume the first argument is a React module, if `tagName` is undefined, the first argument is a options object.

Reply:
ill give it a try! thanks @justinfagnani